php82-moodle-unit/Dockerfile

```diff
@@ -81,8 +83,20 @@ COPY conf.json /var/lib/unit/
 
 COPY xx-moodle.ini /etc/php82/conf.d/xx-moodle.ini
 
+# S6 & cron management
+ADD https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-noarch.tar.xz /tmp
+RUN tar -C / -Jxpf /tmp/s6-overlay-noarch.tar.xz
+ADD https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-x86_64.tar.xz /tmp
+RUN tar -C / -Jxpf /tmp/s6-overlay-x86_64.tar.xz
+
+COPY cron.sh /usr/local/bin/cron.sh
+RUN chmod +x /usr/local/bin/cron.sh
```

Comment:
could be done before file copy - initially needs +x attribute

Reply:
Updated in last commit